src/main/java/com/tg5/service/MessageProducer.java

```diff
@@ -0,0 +1,16 @@
+package com.tg5.service;
```

Comment:
You can remove this file and inject jmsTemplate straight to LowBalanceCronAlert  and use it there.

Reply:
Removed!